### DISCUSSION TITLE
Error with installation of mamba

### DISCUSSION THREAD

**Discussion by xchengood** [idaholab/moose#24089] [Q&A Getting Started] [answered]
Hi,

Error is below. Thank you for your help.

<img width="909" alt="Screen Shot 2023-04-17 at 9 39 46 AM" src="https://user-images.githubusercontent.com/74226455/232520411-23508065-78db-41f2-b7e5-55e9d294812a.png">

**Comment by GiudGiud** [comment, marked as answer]
Hello

Seems you do not have permissions to this directory.
Can you either:
- make a new directory and install everything there
- change the permissions to the directory

Guillaume

**Reply by milljm** [reply]
Seeing that you already have `(base)` prefixed in your prompt, it would appear you already have Conda installed and initialized.
Installing a second Conda instance is not a good idea.

The error you are receiving is some other issue entirely however.
